github/github.go

```diff
@@ -24,12 +24,14 @@ func (c *Client) SetToken(token string) {
 	c.client = c.client.WithAuthToken(token)
 }
 
-func (c *Client) ListTagsForRepository(ctx context.Context, repo svcmodel.GithubRepository, limit int) ([]svcmodel.GitTag, error) {
+func (c *Client) ListTagsForRepository(ctx context.Context, repo svcmodel.GithubRepository) ([]svcmodel.GitTag, error) {
+	// Last 30 tags are fetched when no options are provided
```

Comment:
Love the comment, explanation and link to their docs!

But consider simply putting max value here, i.e. `100`.